documentation/modules/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -7,7 +7,31 @@
 
 The following resources will created by the Cluster Operator in the Kubernetes cluster:
 
+.Shared resources
+
+`_cluster-name_-cluster-ca`:: Secret with the Cluster CA used to encrypt the cluster communication.
+`_cluster-name_-cluster-ca-cert`:: Secret with the Cluster CA public key. This key can be used to verify the identity of the Kafka brokers.
+`_cluster-name_-clients-ca`::  Secret with the Clients CA private key used to sign user certiticates 
+`_cluster-name_-clients-ca-cert`:: Secret with the Clients CA public key. This key can be used to verify the identity of the Kafka users.
+`_cluster-name_-cluster-operator-certs`:: Secret with Cluster operators keys for communication with Kafka and ZooKeeper.
+
+.Zookeeper nodes
+
+`_cluster-name_-zookeeper`:: StatefulSet which is in charge of managing the ZooKeeper node pods.
+`_cluster-name_-zookeeper-_idx_`:: Pods created by the Zookeeper StatefulSet.
+`_cluster-name_-zookeeper-nodes`:: Headless Service needed to have DNS resolve the ZooKeeper pods IP addresses directly.
+`_cluster-name_-zookeeper-client`:: Service used by Kafka brokers to connect to ZooKeeper nodes as clients.
+`_cluster-name_-zookeeper-config`:: ConfigMap which contains the ZooKeeper ancillary configuration and is mounted as a volume by the ZooKeeper node pods.
+`_cluster-name_-zookeeper-nodes`:: Secret with ZooKeeper node keys.
```

Comment:
mention why the same name in case people think it's a typo?

Reply:
I think this is fine. People will see the difference.